detail what the webhooks actually do (#4)
heron-data/docs: docs/webhooks.md

Where:

-   `topic` is the topic of this webhook in the format `<resource>.<event>`. Currently we support webhooks for the following topics:

>     -   `end_user.processed`, triggered when asynchronous processing of an end user has finished and EndUser status set to "processed". Processing happens after the EndUser status is set to "ready", which initiates processing

Reviewer: Bit redundant:
```suggestion
    -   `end_user.processed`, triggered when asynchronous processing of an end user has finished. Processing happens after the EndUser status is set to "ready", which initiates processing
```

    -   `end_user.reviewed`, triggered when an underwriter / Heron has manually reviewed a company and set the EndUser status to "reviewed"

>     -   `end_user.transactions_updated`, triggered 10 minutes after any fields on a transaction (e.g., feedback on category) has changed

Reviewer: ```suggestion
    -   `end_user.transactions_updated`, triggered 10 minutes after the latest change to any transaction (e.g. when giving feedback) for an end user.
```

Where:

* `topic` is the topic of this webhook in the format `<resource>.<event>`. Currently we support webhooks for the following topics:

>     * `end_user.processed`, triggered when asynchronous processing of an end user has finished and EndUser status set to "processed". Processing happens after the EndUser status is set to "ready", which initiates processing

Reviewer: Editing / reducing redundancy:
```suggestion
    * `end_user.processed`, triggered when asynchronous automated processing of an end user has finished. Processing is started after the EndUser status is set to "ready".
```

    * `end_user.reviewed`, triggered when an underwriter / Heron has manually reviewed a company and set the EndUser status to "reviewed"

>     * `end_user.transactions_updated`, triggered 10 minutes after any fields on a transaction (e.g., feedback on category) has changed

Reviewer: Clarifying
```suggestion
    * `end_user.transactions_updated`, triggered within 10 minutes of the last change on transactions for a given end user (e.g. after feedback on category).
```
